docs/sources/whatsnew/whats-new-in-v8-2.md

```diff
@@ -1,42 +1,51 @@
+
+Grafana 8.2 includes a number of important community contributions including support for OAuth role mapping with GitLab accounts ([#30025](https://github.com/grafana/grafana/pull/30025)), a new wide-to-long function ([#38670](https://github.com/grafana/grafana/pull/38670)) included in the [prepare time series transformation](https://grafana.com/docs/grafana/latest/panels/transformations/types-options/#prepare-time-series). A number of additions to the Azure Monitor data source were submitted, including a overview dashboard([#38801](https://github.com/grafana/grafana/pull/38801)), and support for parsing numeric fields in the Azure Resource Graph([#38728](https://github.com/grafana/grafana/pull/38728)). Contributions also included [regular-expression based value mapping](https://grafana.com/docs/grafana/next/panels/value-mappings/#map-a-regular-expression) ([#38931](https://github.com/grafana/grafana/pull/38931)) and improvements to our systemd unit for Grafana installations ([#38109](https://github.com/grafana/grafana/pull/38109)). This list is by no means exaustive or comprehensive and we greatly appreciate all the contributions submitted for inclusion in Grafana.
+
 ## Accessibility
 
-We’ve taken our first, measured but important steps towards improving the accessibility of Grafana, with much needed tweaks to keyboard navigation, accessibility labeling for UI elements for Grafana viewers and an accessibility statement (link coming soon) for Grafana laying out what you can expect from a Grafana focused on making our project accessible to all.
+We’ve taken our first, measured but important steps towards improving the accessibility of Grafana, with much needed tweaks to keyboard navigation, accessibility labeling for UI elements for Grafana viewers and an [accessibility statement](https://grafana.com/accessibility/) for Grafana laying out what you can expect from a Grafana focused on making our project accessible to all.
 
-## Grafana 8 Alerting
+## Dashboards 
 
-We’ve continued to bolster the new, unified alerting system which we launched in Grafana 8. This release includes significant improvements including an AlertManager notification panel allowing users to explore alerts via a dashboard panel. We’ve also added custom grouping for alert manager notifications and a number of small but significant changes to improve creation editing and managing alert rules. Please see the [alerting documentation](https://grafana.com/docs/grafana/latest/alerting/unified-alerting/) for more details and information on enabling the unified alerting in your instance of Grafana.
+The biggest change to dashboards in Grafana 8.2 is the inclusion of a configurable fiscal year in the time tange controls. quarters as time ranges, which can be helpful for business-focused and executive dashboards as well as (https://grafana.com/docs/grafana/latest/dashboards/time-range-controls/) for more information, please see the [time range controls documentation].
```

Comment:
I think something got dropped. Pulled this from the WIP draft:
```suggestion
The biggest change to dashboards in Grafana 8.2 is the inclusion of a configurable fiscal year in the time picker. This option enables fiscal quarters as time ranges, which can be helpful for business-focused and executive dashboards in addition to many other common use cases. Please see the [time range controls documentation].(https://grafana.com/docs/grafana/latest/dashboards/time-range-controls/) for more information.
```

Reply:
Thanks for the catch - I was indeed copy/pasting from multiple sources.